packages/Liquid-Core.package/LiquidModel.class/instance/buildButtonPrefabWith..st

```diff
@@ -0,0 +1,9 @@
+accessing
+buildButtonPrefabWith: aBuilder
```

Comment:
Pebble: Die Methoden in LiquidModel werden meines Wissens nach nicht mehr genutzt. Könnten also im Prozess dieser PR gelöscht werden, oder später irgendwann

Reply:
Ich habe jetzt LiquidModel (und gleich auch LiquidMorph) komplett entfernt, findest du das ne gute Lösung? Die UI-Klassen erben dann direkt von Model

Reply:
Die Funktionalität von `LiquidModel` betrachtend ist das auf jeden Fall ein logischer Schritt, den würde ich so approven. Die UI-Klassen werden dadurch aber "all over the place" verteilt, wir sollten also vermutlich demnächst ein neues Package dafür machen - das war ja sowieso geplant, oder?

Reply:
Ja, glaube wir wollen demnächst in mehreere Packages splitten